docs/authoring.md

```diff
@@ -0,0 +1,108 @@
+# Authoring Components
+
+## Overview
+C#/WinRT provides support for authoring Windows Runtime components. You can write a library in C#, and use C#/WinRT's source generator to get a winmd that any WinRT compatible language can use. For example, a library written in C# can be used by a C++ program, via C#/WinRT and C++/WinRT, with just a few tweaks to the C++ project.
+
+
+## Authoring the C# Component
+To create a library, select the Class Library (.NET Core) template in Visual Studio. 
+Make sure the `<TargetFramework>` for the project is `net5.0-windows10.0.19041.0`. 
+And add the following to your library's project file:
+```
+  <PropertyGroup>
+    <!-- The Windows SDK metadata will need to update as new versions release -->
+    <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
+    <CsWinRTComponent>true</CsWinRTComponent>
+    <CsWinRTEnableLogging>true</CsWinRTEnableLogging>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <GeneratedFilesDir Condition="'$(GeneratedFilesDir)'==''">$([MSBuild]::NormalizeDirectory('$(MSBuildProjectDirectory)', '$(IntermediateOutputPath)', 'Generated Files'))</GeneratedFilesDir>
+  </PropertyGroup>
+```
+And don't forget to include a `PackageReference` to `Microsoft.Windows.CsWinRT`. You can add the nuget feed found in the C#/WinRT README as a nuget source, and install the package in Visual Studio via the `Manage NuGet Packages for Solution` option. 
+
+
+## Known Authoring Issues
+1. There are some programs you could write as/in your component that aren't available in the Windows Runtime. 
+We are working on implementing diagnostics in our tool that will catch these errors before a winmd is generated for your component.
+
+2. Not all C# types have been mapped to runtime types, but we are working on completing this coverage. 
+

```

Comment:
3. Composable support

Reply:
I added "composable type support is still in progress" -- do you have something different you want to say here?

Reply:
That is good.